base/src/main/java/com/smartdevicelink/managers/screen/SoftButtonReplaceOperation.java

```diff
@@ -30,14 +31,16 @@ class SoftButtonReplaceOperation extends Task {
     private final SoftButtonCapabilities softButtonCapabilities;
     private final CopyOnWriteArrayList<SoftButtonObject> softButtonObjects;
     private String currentMainField1;
+    private Boolean isGraphicSupported;
 
-    SoftButtonReplaceOperation(ISdl internalInterface, FileManager fileManager, SoftButtonCapabilities softButtonCapabilities, CopyOnWriteArrayList<SoftButtonObject> softButtonObjects, String currentMainField1) {
+    SoftButtonReplaceOperation(ISdl internalInterface, FileManager fileManager, SoftButtonCapabilities softButtonCapabilities, CopyOnWriteArrayList<SoftButtonObject> softButtonObjects, String currentMainField1, Boolean isGraphicSupported) {
         super("SoftButtonReplaceOperation");
         this.internalInterface = new WeakReference<>(internalInterface);
         this.fileManager = new WeakReference<>(fileManager);
         this.softButtonCapabilities = softButtonCapabilities;
         this.softButtonObjects = softButtonObjects;
         this.currentMainField1 = currentMainField1;
+        this.isGraphicSupported = isGraphicSupported;
```

Comment:
Can we refactor this to be `isDynamicGraphicSupported`? I just think in the future it will be much clearer what this actually means.